docs/virtual_machines/disks_and_volumes.md

```diff
@@ -131,6 +133,18 @@ as a `cdrom` device to the VM:
           persistentVolumeClaim:
             claimName: mypvc
 
+### filesystems
+A `filesystems` disk will expose the volume as a filesystem to the VM.
```

Comment:
This should be: "A `filesystem` device"

 it's a _device_ not a _disk_. Also, I think `filesystems`  it should be in singular

Reply:
Addresed.